lib/sequent/core/helpers/helpers.rb

```diff
@@ -1,3 +1,4 @@
+require_relative 'functions'
```

Comment:
`functions` can be more specific. But also: do we really need this? See previous comments on `Class.const_get`.